src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -87,20 +87,26 @@ class TargetNotExported(Exception):
     """Indicates a target that was expected to be exported is not."""
 
 
-class NoOwnerError(Exception):
-    """Indicates an exportable target has no owning exported target."""
+class InvalidEntryPoint(Exception):
+    """Indicates that a specified binary entry point was invalid."""
 
 
-class AmbiguousOwnerError(Exception):
-    """Indicates an exportable target has more than one owning exported target."""
+class OwnershipError(Exception):
+    """An error related to target ownership calculation."""
 
+    def __init__(self, msg: str):
+        super().__init__(
+            f"{msg} See https://www.pantsbuild.org/v2.0/docs/python-setup-py-goal for "
```

Comment:
This is likely to become stale. But it's fine given the nature of 2.0.

We'll want to audit for the string "v2.0" as soon as we switch the default version of the docs.

Reply:
Yeah, there is some general tooling we'll need to build around the docs (e.g., finding broken links). Basic stuff that readme.com doesn't provide yet.